building/build-debs/homeworld-prometheus-node-exporter/.gitignore

```diff
@@ -0,0 +1,5 @@
+go-bin-*
+prometheus-*.tar.xz
+promu-*.tar.xz
+gopath
+./bin/
```

Comment:
.gitignore doesn’t work that way—you mean `/bin/` (and you possibly mean to have a leading `/` on most .gitignore entries in the whole repository?).